apis/python/setup.py

```diff
@@ -326,8 +326,10 @@ def run(self):
     zip_safe=False,
     setup_requires=["pybind11"],
     install_requires=[
-        # Tracked in https://github.com/single-cell-data/TileDB-SOMA/issues/1785
-        "anndata != 0.10.0",
+        # Temporary for 1.15.0rc series -- avoid RC versions of these packages
+        "anndata==0.10.9",
```

Comment:
```suggestion
        "anndata>=0.10.1,<0.11",
```

I think this will leave downstream users more flexibility